### PR TITLE
Cookoff - Use `getShotInfo` for tracer detection

### PR DESCRIPTION
**Pull request by johnb432**
**When merged this pull request will:**
- Title.
- Requires #9758.

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/cookoff/functions/fnc_smoke.sqf [hunk 1]
```diff
@@ -31,7 +30,7 @@ private _effects = [_smokeBarrel];
 {
     private _position = [0, -2, 0];
 
-    if (_x isNotEqualTo "#noselection") then {
+    if (_x != "#noselection") then {
```
Comment (rautamiekka) [inline addons/cookoff/functions/fnc_smoke.sqf:33]: That's a totally pointless change, all it does is worsen things.

Comment (johnb432) [reply addons/cookoff/functions/fnc_smoke.sqf:33]: a) #9758 
b) No it doesn't.

File: addons/cookoff/functions/fnc_handleDamageBox.sqf [hunk 2]
```diff
@@ -32,30 +32,12 @@ if !(_hitPoint == "" && {_damage > 0.5}) exitWith {}; // "" means structural dam
 private _ammoConfig = _ammo call CBA_fnc_getObjectConfig;
 
 // Catch fire when hit by an explosive or incendiary round
-if ((getNumber (_ammoConfig >> "explosive") >= 0.5) || {getNumber (_ammoConfig >> QEGVAR(vehicle_damage,incendiary)) > random 1}) then {
+if (
+    (getNumber (_ammoConfig >> "explosive") >= 0.5) ||
+    {getNumber (_ammoConfig >> QEGVAR(vehicle_damage,incendiary)) > random 1} ||
+    {_damage * 0.05 > random 1 && {_ammo getShotInfo 4}} // there is a small chance of cooking a box off if it's shot by tracer ammo
```
Comment (johnb432) [inline addons/cookoff/functions/fnc_handleDamageBox.sqf:38]:
```suggestion
    {_damage * 0.05 > random 1 && {_ammo getShotInfo 4}} // There is a small chance of cooking a box off if it's shot by tracer ammo
```

**Comment by johnb432** [comment]
`_ammo getShotInfo 4` can't be used, as `_ammo` is the classname and not the object. We could try to use the `HitPart` EH, but locality weirdness with that EH is an issue, since `HitPart` triggers even if `allowDamage` is false, meaning we'd need to check it manually and given it's local command, we'd need to send events over the network every time.
TLDR: not worth it.
